apps/HelloAndroidCamera2/TODO.txt

```diff
@@ -0,0 +1,8 @@
+- fix proper startup: resolutions, yuck
```

Comment:
Thanks for including a TODO! A little more verbosity would be good in explaining what needs to be done.